kratos/processes/levelset_convection_process.h

```diff
@@ -692,11 +697,11 @@ class LevelSetConvectionProcess
             const double fraction = std::abs(numerator) / (denominator + epsilon);
 
             if (mIsBfecc){
-                mLimiter[i_node] = 1.0 - std::pow(fraction, power_bfecc);
+                mLimiter[i_node] = 1.0 - std::pow(fraction, mPowerBfeccLimiter);
```

Comment:
One question: did you try without the limiter for BFECC ? I heard from other people in Altair that they were quite happy with their results without having the limiter correction... Maybe it makes sense to have it as optional

Reply:
Without a limiter, BFECC disturbs the monotonicity of the underlying scheme. Having now the "acuteness" in the options, one can simply set a very large value to reduce the effect of the limiter (nearly deactivate it).

Reply:
Excuse my ignorance, but does the original equation (no BFECC and default "levelset_convection_element_simplex") satisfy the Discrete Maximum Principle? I thought it did not and that was why you could see over/under-shots. I am not very familiar with all these ideas, so any help is very welcome.

Reply:
The standard SUPG does not comply with DMP. The cross-wind stabilization improves it, however, it still is not monotonicity preserving (not full compliance with DMP). 
The point is that using BFECC without a limiter severely worsens the over/under-shoots.

Reply:
Thanks for the clarification 👍